Remove inaccurate message from image block

aladin002dz opened the pull request:
## Description
Remove an inaccurate message from the image block, to make it the
same as the other media files: audio and video. Fixes #20682


## How has this been tested?
npm run test


## Screenshots 
before:
![before](https://user-images.githubusercontent.com/26669835/76703231-79bece80-66d0-11ea-86c8-39282046c910.png)

after:
![after](https://user-images.githubusercontent.com/26669835/76703244-9b1fba80-66d0-11ea-85fe-15995cb06990.png)


## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [x] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [x] I've included developer documentation if appropriate. <!-- Handbook: https://developer.wordpress.org/block-editor/ -->
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. <!-- React Native mobile Gutenberg guidelines: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/native-mobile.md -->

youknowriad commented on package-lock.json:
```diff
@@ -19794,7 +19794,7 @@
 				"nopt": {
 					"version": "4.0.1",
-					"resolved": false,
+					"resolved": "https://registry.npmjs.org/nopt/-/nopt-4.0.1.tgz",
```
I believe the package-lock change is probably not needed in this PR, right?

aladin002dz replied:
> I believe the package-lock change is probably not needed in this PR, right?

yeah, right, this was made automatically, by "npm install" I guess.

youknowriad reviewed (commented):
This seems to work as intended. Can we remove the package-lock changes and ship?

aladin002dz commented:
> This seems to work as intended. Can we remove the package-lock changes and ship, sure!

> This seems to work as intended. Can we remove the package-lock changes and ship?

yeah, sure!

aladin002dz committed:
Remove inaccurate message from image block

Remove an inaccurate message from the image block, to make it the
same as the other media files: audio and video. Fixes #20682

aladin002dz commented:
> This seems to work as intended. Can we remove the package-lock changes and ship?

Done! 
$ git pull upstream master (to reset all files and update local repo)
$ git checkout <branch fix>
$ git reset --hard HEAD^
remake change quickly, commit and push branch

youknowriad reviewed (approved):
LGTM 👍

youknowriad commented:
Congrats for your first contribution.
